src/callbacks_stage/subcell_limiter_idp_correction_2d.jl

```diff
@@ -5,7 +5,7 @@
 @muladd begin
 #! format: noindent
 
-function perform_idp_correction!(u, dt, mesh, equations, dg, cache)
+function perform_idp_correction!(u, dt, mesh::AbstractMesh{2}, equations, dg, cache)
```

Comment:
No, it doesn't. I just thought because it works for all mesh types that support subcell limiting. But you're right. It probably makes more sense to just list all mesh types and add whenever I add the support for another one.